opendebates/tests/test_navigation.py

```diff
@@ -52,9 +52,9 @@ def test_search_term_is_sticky(self):
 
     def test_search_term_inserts_into_sort_form(self):
         rsp = self.client.get(self.search_url + "?q=something+ducks")
-        form_html = ('\<form action="{search}" method="GET" '
-                     'class="form-inline"\>\W+'
-                     '\<input type="hidden" name="q" '
-                     'value="something ducks"\>'.format(
+        form_html = (r'\<form action="{search}" method="GET" '
```

Comment:
I'm not sure why the "<" are being preceded by "\" in this regex originally. I think we don't want to do that, because "<" by itself is a normal character in a regex and does not need to be escaped.  In both str and raw string literals, because "\<" is not a recognizable special escaped char (like "\n" would be), Python will just keep both chars in the string, as if we had written "\\" + "<", and that would seem to mean that this regex could only match HTML that had an actual backslash at those positions.

Reply:
Makes sense.